app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -20,3 +20,14 @@
 .gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
   border-color: govuk-colour("white");
 }
+
+@include govuk-media-query($until: tablet) {
+  .govuk-breadcrumbs__list-item:not(:last-child):not(:first-child) {
+    display: none;
+  }
+
+  .govuk-breadcrumbs__link {
+    padding-top: 14px;
+    padding-bottom: 14px;
```

Comment:
Now the padding is on the links (correct) the links are a lot bigger. Because they're inline elements, the padding 'leaks' out of the containing element, see below.

<img width="318" alt="Screenshot 2020-02-13 at 15 19 23" src="https://user-images.githubusercontent.com/861310/74449369-6243ba00-4e74-11ea-8397-7262d07059d0.png">

Can you attach a screenshot of this behaviour in context on a GOV.UK page? I'm concerned about the overlap. We may need to apply more padding to the containing element to compensate for this.

Reply:
Can we make the links `inline-block`?

Reply:
That's an option, but it's worth checking what effect that has on the chevron positioning. I tried it initially when the two breadcrumbs covered different numbers of lines and it went a bit wonky.

Reply:
have added margin around the breadcrumb
![Screenshot 2020-02-14 at 11 35 07](https://user-images.githubusercontent.com/54625020/74527618-15b4b900-4f1e-11ea-8e8f-81d03fafd81e.png)

Reply:
If you want a bigger touch size but a consistent focus style you can try using an absolutely positioned element pseudo:

https://govuk-publishing-components-1290.glitch.me/

Reply:
☝️ that's cool.

Reply:
Nice one Nick. Have incorporated to breadcrumb